tests/example/test_regular_examples.py

```diff
@@ -20,7 +21,7 @@
     mosmix_forecasts,
     observations_sql,
     observations_stations,
-    observations_station_gaussian_model,
+    pytest.param(observations_station_gaussian_model, marks=pytest.mark.skipif(not IS_LINUX, reason="stalls in CI")),
     dwd_describe_fields,
```

Comment:
Can't we just create this list at runtime, instead of needing all sorts of silly things to make it work at compilation time?

Reply:
Hint: I think the solution is to make `EXAMPLES` a fixture as well.

Reply:
Other than this, what's actually the purpose of this test case definition? Only to invoke `.main()` on some callable?

```python
@pytest.mark.cflake
@pytest.mark.parametrize("example", EXAMPLES)
def test_regular_examples(example):
    assert example.main() is None
```

I think this is slightly underengineered. Let's just remove `observations_station_gaussian_model` from `EXAMPLES`, and invoke it separately, no?

Reply:
Wouldn't this be way more explicit?
```python
@pytest.mark.cflake
def test_gaussian_example(is_linux):
    if not is_linux:
        raise pytest.skip("stalls in CI")
    assert observations_station_gaussian_model.main() is None
```

Reply:
We just want to make sure that the tests are running (by given code), like with the Jupiter notebook.

Reply:
Sure. I would have expected that all examples are enumerated from the `examples` directory, but now that we enumerate them explicitly, I see no harm in breaking out a single outlier, in order to skip it procedurally, without needing to fight against `pytest.mark.skip`, which is primarily intended to be used as a decorator.

Reply:
On the other hand, there is also [pytest-platform-markers](https://github.com/The-Compiler/pytest-platform-markers). Can this be leveraged within this problem scope? Hm. I still think I would prefer the explicit variant.